packages/salesforcedx-vscode-soql/test/vscode-integration/editor/htmlUtils.test.ts

```diff
@@ -16,7 +16,7 @@ describe('html utilities', () => {
   const pathToLwcDist = SOQL_BUILDER_UI_PATH;
   const html = `
   <!-- CSP TAG -->'
-  <script src="./0.app.js"></script><script src="./app.js"></script>
+  <script defer="defer" src="./0.app.js"></script><script defer="defer" src="./app.js"></script>
```

Comment:
I think defer is a boolean attribute, so `<script defer src="./0.app.js">` should be sufficient.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script

Reply:
You are correct, however the output in the html is in the format seen here `defer="defer"` so we need to match for that.